tuneme/templates/new/core/language_block.html

```diff
@@ -5,7 +5,7 @@
   {% trans "Language" %}:
   {% get_current_language as LANGUAGE_CODE %}
   {% for language in languages %}
-    <a href="{% url 'locale_set' language.locale %}?next={{request.path}}?{{ request.GET.urlencode }}" class="languages__item {% if LANGUAGE_CODE == language.code %} languages__item--active{% endif %}">{{ language }}</a>
+    <a href="{% url 'locale_set' language.locale %}?next={{request.path}}?{{ request.GET.urlencode }}" class="languages__item {% if LANGUAGE_CODE == language.locale %} languages__item--active{% endif %}">{{ language }}</a>
```

Comment:
@saeedmb8 can you comment on this? I had to use `locale` instead of `code`, but I'm not sure where code came from? It was just empty on my side.